content/docs/user-guide/interactivity/physics/nvidia-physx/simulated-bodies.md

```diff
@@ -0,0 +1,62 @@
+---
+description: ' PhysX Simulated Bodies. '
+title: PhysX Simulated Bodies
+weight: 300
+---
+
+`AzPhysics::SimulatedBody` is a base type from which many PhysX components inherit:
++ **[PhysX Rigid Body](/docs/user-guide/components/reference/physx/rigid-body-physics/)**
++ **[PhysX Character Controller](/docs/user-guide/components/reference/physx/character-controller/)**
++ **[PhysX Ragdoll](/docs/user-guide/components/reference/physx/ragdoll/)**
+
+The interface for `AzPhysics::SimulatedBody` includes various common operations such as:
+- Handling collision and trigger events
+- Getting the Entity Id for the **Entity** to which the **Simulated Body** is attached
+- Getting the **Entity** **Transform**
+- Getting the **Entity** **AABB**
+- Performing a raycast against a particular **Simulated Body**
+
+This functionality is also exposed via the **SimulatedBodyComponentRequestsBus**.
+
+## Raycast
```

Comment:
Maybe the section could be called "Simulated Body Raycast"?

Reply:
I think it should be fine since it's inside the document on Simulated Bodies, and it explains that there are also scene query raycasts and links to those (and vice versa from the other page)